### DISCUSSION TITLE
[ideas] tar_reduce - ability to aggregate/summarize multiple targets

### DISCUSSION THREAD

**Discussion by collinschwantes** [ropensci/targets#1260] [Ideas]
### Help

- [X] I understand and agree to <https://books.ropensci.org/targets/help.html>.

### Description

This idea is similar to whats discussed in #987.   Lets say I have a set of targets that I need to combine into a single data frame. They all start with "xyz_". Ideally, I could run something like `tar_reduce(name = combined_df, values = starts_with("xyz_"), command = rbind)` which would give me my desired data frame. Any ideas on how to do this? 

The example below seems to work but maybe there is a better solution?

```
test <- tar_plan(

    tar_target(mtcars1, mtcars),
    tar_target(mtcars2, mtcars),
    tar_target(mtcars3, mtcars),
    tar_target(iris1, iris)

)

list(
    test,
    tar_combine(combined, tar_select_targets(test, starts_with("mtcars")))
)
```

**Comment by wlandau** [comment]
`tar_combine()` is the canonical "reduce" solution in the `targets` ecosystem. At a glance, your code sketch looks right.
